tasks/messages/types/ipldsch_minima.go

```diff
@@ -10,14 +10,14 @@ import (
 )
```

Comment:
gofmt'd these files and the others in this dif.

Reply:
We should add that as a step in the generation task